pkgs/development/tools/boomerang/default.nix

```diff
@@ -1,38 +1,43 @@
-{ stdenv, fetchgit, cmake, expat, qt5, boost }:
+{ stdenv, fetchFromGitHub, cmake, qtbase }:
 
 stdenv.mkDerivation rec {
   name = "boomerang-${version}";
-  version = "0.3.99-alpha-2016-11-02";
+  version = "0.4.0-alpha-20180118";
```

Comment:
Could you keep hyphens in the date? (https://nixos.org/nixpkgs/manual/#sec-package-naming documents that `YYYY-MM-DD` is the preferred format)

Why are you changing `0.3.99` to `0.4.0`? The latest tag is `v0.3.99-alpha`, so leaving it as is seems appropriate.

Reply:
It's the version the code gives itself currently: https://github.com/ceeac/boomerang/blob/dc53653f2374676c07296b4c51eed4ac4eeba127/cmake-scripts/boomerang-version.cmake#L11

Happy to conform with date, wasn't sure how to handle this.
My understanding is that `2018-01-18` (nothing else) is preferred, but didn't want to "break" upgrades by anyone currently having it installed.  Will change as you suggest.